packages/fxa-admin-panel/package.json

```diff
@@ -61,7 +61,7 @@
     "@babel/core": "^7.10.3",
     "@rescripts/cli": "0.0.16",
     "@storybook/addon-actions": "^5.3.19",
-    "@storybook/addon-links": "^6.2.9",
+    "@storybook/addon-links": "^6.2.8",
```

Comment:
This needs to be reverted back to 5.3.19

Reply:
I'm confused how dependabot managed to go from 6.2.8 to 6.2.9 when [the my previous revert was down to 5.3.19](https://github.com/mozilla/fxa/pull/8392). I think 6.2.8 will still be broken

Reply:
Ohhh, I see... my revert was for `@storybook/addons` and this is for `@storybook/addon-links` which somehow got upgraded earlier?